timers.js

```diff
@@ -1,4 +1,48 @@
+//Variables seconds & timerID
+var seconds = 0;
+var timerIDSelector;
+var stopWatch;
 
+//Update the timer
+function updateTime(){
+  seconds++;
+  timerIDChange();
+}
 
+//Stop the timer
+function stopTime(){
+  clearInterval(stopWatch);
+}
 
+//Change the text in ID timer
+function timerIDChange(){
+  timerIDSelector = document.querySelector("#timer");
```

Comment:
You might as well define `timerIDSelector` when you instantiate it at the top of the document since its value doesn't depend on user interaction. Save yourself an extra line of code.